src/js/main.js

```diff
@@ -1,8 +1,8 @@
 import React, {Component} from 'react';
 import ReactDOM from 'react-dom';
 import {renderedTimeline} from './services';
-import {TimelineButton, User, TimelineComp} from './components';
-import {getHomeTimeline, getUserTimeline} from './services';
+import {TimelineButton, User, TimelineComponent} from './components';
+import {getHomeTimeline, getUserTimeline, getFilterTimeline} from './services';
```

Comment:
with the timelines being different, any particular reason why you have them in main.js?